systems/plants/CMakeLists.txt

```diff
@@ -1,20 +1,21 @@
 
+
+include_directories (${CMAKE_SOURCE_DIR}/util tinyxml)
+
 # IMPORTANT NOTE!!
 # matlab has it's own boost libraries.  DO NOT let any mex file depend
 # on the system boost (directly nor indirectly), or you're asking for trouble.
 find_package( Boost COMPONENTS filesystem system regex)
```

Comment:
because i'm supporting the URDFRigidBodyManipulator class until I can sit down w/ @patmarion to purge it.